### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by DenisVuyka**
refs #1357
refs #400

### CODE REVIEW

File: ng2-components/ng2-alfresco-documentlist/README.md [hunk 1]
```diff
@@ -437,18 +452,53 @@ For a full list of available `format` values please refer to [DatePipe](https://
...
+    <!-- template body -->
+</template>
+```
+
+The format of naming is `let-VARIABLE_NAME="$implicit"` where `VARIABLE_NAME` is the name of the variable you want binding template data context to.
+
+Getting a cell value from the underlying DataTableAdapter:
+
+```ts
+context.data.getValue(entry.row, entry.col);
```
Comment (magemello) [inline ng2-components/ng2-alfresco-documentlist/README.md]: which one is correct?
- context.data.getValue(entry.row, entry.col);
- entry.data.getValue(entry.row, entry.col);

File: ng2-components/ng2-alfresco-documentlist/README.md [hunk 2]
```diff
@@ -437,18 +452,53 @@ For a full list of available `format` values please refer to [DatePipe](https://
...
+context.row.getValue('createdByUser.displayName')
+```
+
+_You may want using **row** api to get raw value access. 
+Use **data** api to get values with post-processing, like datetime/icon conversion._
+
+Final example:
+
+```html
+<content-column title="Name" key="name" sortable="true" class="full-width ellipsis-cell">
```
Comment (magemello) [inline ng2-components/ng2-alfresco-documentlist/README.md]: Can I use something like that? If so can you add it to the examples?

```html
<content-column title="Name" key="name" sortable="true" class="full-width ellipsis-cell">
    <template let-entry="$implicit">
        <span>Hi! {{entry.row.getValue('createdByUser.displayName')}}</span>
        <span>Hi! {{entry.row.getValue('name')}}</span>
    </template>
</content-column>
```

File: ng2-components/ng2-alfresco-documentlist/README.md (hunk 1 above)
Comment (DenisVuyka) [inline ng2-components/ng2-alfresco-documentlist/README.md]: First, as the step above it demos naming property as `context`

File: ng2-components/ng2-alfresco-documentlist/README.md (hunk 2 above)
Comment (DenisVuyka) [inline ng2-components/ng2-alfresco-documentlist/README.md]: Yes, you can.

**Pull request merged**